README.md

```diff
@@ -173,6 +173,37 @@ const styles = StyleSheet.create({
 
 Aphrodite will ensure that the global `@font-face` rule for this font is only inserted once, no matter how many times it's referenced.
 
+# Descendant selectors
+
+Descendant rules (i.e. a rule like `.parent:hover .child { ... }` in CSS) are created by adding values to your styles indicating the styles to apply to the child. To distinguish the styles from normal CSS keys, the keys start with a `>>`, like `'>>child'`.
+
+```js
+const styles = StyleSheet.create({
+    parent: {
+        ":hover": {
+            ">>my_child": {
```

Comment:
personal preference: use `myChild` rather than `my_child` just because I don't like `snake_case` in my JavaScript (it's common to lint against it) and I'd prefer examples to use a style that's generally acceptable for JavaScript object property names.